### PR TITLE
Fixed some nits

### PR DESCRIPTION
**Pull request by Honry**
***
<a href="https://pr-preview.s3.amazonaws.com/Honry/web-nfc/pull/525.html" title="Last updated on Jan 10, 2020, 8:09 AM UTC (a59d616)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/525/1633cb5...Honry:a59d616.html" title="Last updated on Jan 10, 2020, 8:09 AM UTC (a59d616)">Diff</a>

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
Thanks for catching those nits. Some overlap with https://github.com/w3c/web-nfc/pull/524.
I'll wait for @kenchris and @zolkis reviews on https://github.com/w3c/web-nfc/pull/524 before merging those if that's OK.

**Review by zolkis — APPROVED** [review]

**Comment by Honry** [comment]
@zolkis, @beaufortfrancois, I've rebased it to the top of `gh-pages` branch.

**Review by beaufortfrancois — APPROVED** [review]
Thank you!

**Pull request merged**
